### DISCUSSION TITLE
Verbose Mode

### DISCUSSION THREAD

**Discussion by christina-pino** [pcdshub/atef#116] [Ideas]
I was going through the main.py file and looking at the flags included already. I am thinking it may be worth discussing a verbose flag and or other modes of output. Say:
-vv "print all values, not depending on if they pass or fail"
-vd "print including all descriptions of checks"
-va "print everything"
These are spit ball ideas but hoping you get where I am trying to go with this

**Comment by klauer** [comment]
We can definitely customize the output - what's there is just the first thing that came to my mind. Perhaps you could propose what the different output modes would look like by example, and then we can refine from there. 

Don't worry about the actual flags (`-vv`) etc. that `atef check` will see - we'll work that out when we get to writing the change.

**Comment by ZLLentz** [comment]
I'm a fan of having output customization- I particularly think the option to see the descriptions will be useful.
